core/pkg/hostsensorutils/logging_messages.go

```diff
@@ -0,0 +1,10 @@
+package hostsensorutils
+
+// messages used for warnings
+var (
+	failedToGetData                     = "failed to get data"
+	failedToTeardownNamespace           = "failed to teardown namespace"
+	oneHostSensorPodIsUnabledToSchedule = "One host-sensor pod is unable to schedule on node. We will fail to collect the data from this node"
+	failedToWatchOverDaemonSetPods      = "failed to watch over DaemonSet pods"
```

Comment:
```suggestion
	failedToWatchOverDaemonSetPods      = "failed to watch over the host-sensor pods"
```

Reply:
--> Host-Scanner instead of 'host-sensor' (host-sensor word is deprecated :) )

Reply:
I'm fixing it.